packages/token-translator/src/postcss-token-translator.test.ts

```diff
@@ -0,0 +1,99 @@
+import postcss from "postcss";
+
+import plugin from "./postcss-token-translator";
+
+async function run(input: string, output: string, opts = {}) {
+  let result = await postcss([plugin(opts)]).process(input, {
+    from: undefined,
+  });
+  expect(result.css).toEqual(output);
+  expect(result.warnings()).toHaveLength(0);
+}
+
+it("default", async () => {
+  const input = `.a {
+    color: var(--accent);
+}
+`;
+
+  const output = `.a {
+    color: var(--vkui--color_text_accent);
+}
+`;
+
+  await run(input, output);
+});
+
+it("stroke", async () => {
+  const input = `.a {
+    box-shadow: var(--accent);
+    border-color: var(--accent);
+    border: var(--accent);
```

Comment:
Чтобы был более честный тест
```suggestion
    box-shadow: 0 0 2px var(--accent);
    border-color: var(--accent);
    border: 1px solid var(--accent);
```

Reply:
✅